package.json

```diff
@@ -23,10 +23,11 @@
   },
   "dependencies": {
     "electron": "1.4.1",
-    "request": "2.69.0",
+    "electron-edge": "6.5.4",
+    "gpii-windows": "git://github.com/danailbd/windows.git#hst-2017",
     "infusion": "3.0.0-dev.20170830T182157Z.392b2f8",
-    "gpii-windows": "git://github.com/GPII/windows.git#hst-2017",
     "node-jqunit": "1.1.7",
-    "electron-edge": "6.5.4"
+    "request": "2.69.0",
+    "winstrap": "^0.5.11"
```

Comment:
Our standard is to always bind to exact versions of dependencies to avoid injection of security flaws, etc.